static/app/views/performance/browser/webVitals/pageOverview.tsx

```diff
@@ -59,14 +59,16 @@ const LANDING_DISPLAYS = [
 const SAMPLES_COLUMN_ORDER: GridColumnOrder<
   keyof TransactionSampleRowWithScoreAndExtra
 >[] = [
+  {key: 'id', width: COL_WIDTH_UNDEFINED, name: 'Event ID'},
```

Comment:
I preferred when this was just transaction button - should we keep that? Event ID seems a bit confusing if you're unfamiliar with sentry event setup.

Reply:
Hey Abhi, we went back and forth on this a bit in a recent design meeting. The decision we made was to stick with the event id for now, but we'll see if we get any further feedback on this. Thanks!